sdk/openai/openai/src/api/requestOptionDeserializer.ts

```diff
@@ -0,0 +1,17 @@
+import { OperationOptions } from "@azure/core-client";
```

Comment:
maybe we should move this into core-util ?

Reply:
I don't think we have to pull in `core-client` for this. Since `core-client` and `core-client-rest` should never be used together, we can have `core-client-rest` define its own version of the interface with the same name. In time I imagine `core-client` going away or shrinking down considerably as `ServiceClient` will no longer be necessary.